disk/datadog_checks/disk/data/conf.yaml.default

```diff
@@ -98,6 +101,14 @@ instances:
     #   - /dev/sde
     #   - '[FJ]:'
 
+    ## @param include_all_devices - boolean - optional - default: True
```

Comment:
Why is there an uppercase T in `True` here, whereas it's lowercase in the spec & in all other config options?

Reply:
🤔 I am not sure why, the `conf.yaml.default` file is generated automatically by `ddev validate config -s` using the `spec.yaml` file.

Reply:
It's how the config validator works since I added both a `default` and an `example` value:
https://github.com/DataDog/integrations-core/blob/fda6a83d25635c1faae7d18d376947d62053786d/datadog_checks_dev/datadog_checks/dev/tooling/configuration/consumers/example.py#L115-L124
It prints the `repr` of the object (which is `True` since that's how it is written in Python) instead of `true` or `false`, which it does when there is no `default` config.

I am going to change it though, since it seems like elsewhere the `example` and `default` values always match and it can be confusing.

Reply:
I opened #7405 to make the behavior consistent.